### PR TITLE
add zuul support

### PR DESCRIPTION
**Pull request by fredericlepied**
Depends-On: https://github.com/wazo-pbx/sf-jobs/pull/11

### CODE REVIEW

File: integration_tests/suite/helpers/base.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ class BaseIntegrationTest(AssetLaunchingTestCase):
 
     @classmethod
     def setUpClass(cls):
-        super().setUpClass()
+        super(BaseIntegrationTest, cls).setUpClass()
```
Comment (sileht) [inline integration_tests/suite/helpers/base.py]: Why ? This should not be needed if the project is already in python3

File: tox.ini [hunk 2]
```diff
@@ -15,6 +15,22 @@ deps =
...
+
+[testenv:linters]
+skip_install = true
+basepython = python3
+deps = flake8
+       flake8-colors
+       black
+commands =
+  # Don't fail on black failure for now
+  -black --skip-string-normalization --check .
```
Comment (sileht) [inline tox.ini]: If black pass you can remove the `-`

File: tox.ini [hunk 3]
```diff
@@ -36,3 +52,22 @@ deps =
...
+
+[testenv:integration]
+usedevelop = true
+deps = -rintegration_tests/test-requirements.txt
+changedir = integration_tests
+commands =
+    make test-setup
+    make test
+whitelist_externals =
+    sh
```
Comment (sileht) [inline tox.ini]: This is missing:
```
passenv =
    WAZO_TEST_DOCKER_OVERRIDE_EXTRA
    INTEGRATION_TEST_TIMEOUT
```

File: integration_tests/suite/helpers/base.py (hunk 1 above)
Comment (fredericlepied) [reply integration_tests/suite/helpers/base.py]: it was failing in the integration tests...

File: tox.ini (hunk 3 above)
Comment (fredericlepied) [reply tox.ini]: what does it do ?

File: tox.ini (hunk 2 above)
Comment (fredericlepied) [reply tox.ini]: ok

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

File: tox.ini [hunk 4]
```diff
@@ -8,13 +8,31 @@ envlist = py35, pycodestyle, pylint
...
+basepython = python3
 commands =
     pytest --junitxml=unit-tests.xml --cov=wazo_setupd --cov-report term --cov-report xml:coverage.xml wazo_setupd
 deps =
     -rrequirements.txt
     -rtest-requirements.txt
     pytest-cov
 
+[testenv:black]
+basepython = python3
```
Comment (sileht) [inline tox.ini]: basepython is already define in global section.

File: tox.ini [hunk 5]
```diff
@@ -8,13 +8,31 @@ envlist = py35, pycodestyle, pylint
...
+skip_install = true
+deps = black
+commands = black --skip-string-normalization .
+
+[testenv:linters]
+skip_install = true
+basepython = python3
```
Comment (sileht) [inline tox.ini]: basepython is already define in global section.

File: tox.ini [hunk 6]
```diff
@@ -8,13 +8,31 @@ envlist = py35, pycodestyle, pylint
...
+deps = flake8
+       flake8-colors
+       black
+commands =
+  # Don't fail on black failure for now
+  -black --skip-string-normalization --check .
+  flake8
+
 [testenv:pycodestyle]
 basepython = python3
```
Comment (sileht) [inline tox.ini]: basepython is already define in global section.

File: tox.ini [hunk 7]
```diff
@@ -36,3 +54,26 @@ deps =
     pylint
 whitelist_externals =
     sh
+
+[testenv:integration]
+basepython = python3
```
Comment (sileht) [inline tox.ini]: basepython is already define in global section.

**Pull request merged**